main.c

```diff
@@ -1,50 +1,25 @@
 #include <stdio.h>
```

Comment:
code in main.c can be rearranged and moved to separate production and test files to keep this file lighter and simpler.